pkg/operator/operatorclient.go

```diff
@@ -45,7 +45,7 @@ func (c OperatorClient) UpdateOperatorSpec(resourceVersion string, spec *operato
 }
 
 func (c OperatorClient) UpdateOperatorStatus(resourceVersion string, status *operatorv1.OperatorStatus) (*operatorv1.OperatorStatus, error) {
-	original, err := c.Informers.Operator().V1().OpenShiftControllerManagers().Lister().Get("cluster")
+	original, err := c.Informers.Operator().V1().OpenShiftControllerManagers().Lister().Get(globalConfigName)
 	if err != nil {
```

Comment:
These functions need to work on correct object. is `OpenshiftControllerManager` correct object?